### PR TITLE
Implements display and prompt chunk options

### PR DESCRIPTION
**Pull request by gragusa**
This implements what was discussed in #39. 

If you think this is the right way to go I can try adding tests and a doc entry. 

P.S.: I think there is some code replication, but I like to have the flexibility to change how the display is implemented when `display=true`.

### CODE REVIEW

File: src/run.jl [hunk 1]
```diff
@@ -208,6 +209,8 @@ function eval_chunk(chunk::CodeChunk, report::Report, SandBox::Module)
     chunk.result = run_code(chunk, report, SandBox)
     if chunk.options[:term]
         chunks = collect_results(chunk, TermResult())
+    elseif chunk.options[:display] && !chunk.options[:term]
```
Comment (mpastell) [inline src/run.jl]: `!chunk.options[:term]` is not needed here

**Comment by mpastell** [comment]
Thanks! Generally it looks good and thanks for adding the test as well. See my line comment above about adding another formatter.

**Comment by gragusa** [comment]
I have updated `run.jl` as you pointed out in the inline comment.

**Comment by mpastell** [comment]
Thanks,
Can you explain why you'd like to have a separate  `collect_results(chunk::CodeChunk, fmt::DispResult)` method instead of just using the existing ScriptResult/CollectResult ?

**Comment by gragusa** [comment]
This is what I mean when I said "there is some code replications". I was thinking about having more flexibility. After thinking about it, it seems that this flexibility will never be used. I will take care of it.

**Comment by mpastell** [comment]
Looks good, you can always add the flexibility back if there is a use case. I will merge this now, it would be great if you could also had the time to add an entry to the docs but I can also do it myself at some point.

Thanks again!

**Pull request merged**